src/ext/ca/serverca/scasched/scauser.cpp

```diff
@@ -515,7 +515,7 @@ HRESULT ScaUserExecute(
         if (wzDomain && *wzDomain)
         {
             er = ::DsGetDcNameW(NULL, wzDomain, NULL, NULL, NULL, &pDomainControllerInfo);
-            if (HRESULT_FROM_WIN32(er) == RPC_S_SERVER_UNAVAILABLE)
+            if (RPC_S_SERVER_UNAVAILABLE == er)
```

Comment:
MSDN documentation doesn't say anything about this error code. While I agree it looks like it was wrong before, what is this change going to make work now? The comment on line 520 confuses this issue.